ta/pkcs11/src/processing.h

```diff
@@ -124,4 +127,12 @@ enum pkcs11_rc generate_ec_keys(struct pkcs11_attribute_head *proc_params,
 
 size_t ecdsa_get_input_max_byte_size(TEE_OperationHandle op);
 
+enum pkcs11_rc wrap_data_by_symm_enc(struct pkcs11_session *session,
+				     void *data, uint32_t data_sz,
+				     void **out_buf, uint32_t *out_sz);
+
+enum pkcs11_rc unwrap_key_by_symm(struct pkcs11_session *session, void *data,
+				  uint32_t data_sz, void **out_buf,
+				  uint32_t *out_sz);
+
```

Comment:
These should be in "symmetric" section in the file

Reply:
Sure, I will move it there. Didn't notice it while rebasing.